### PR TITLE
115 Cookbook Readme and Recipe TOC

### PR DESCRIPTION
**Pull request by fayvor**
Per [pm/#115](https://github.com/ibm-granite-community/pm/issues/115):
- Add Cookbook README
- Add Recipe Table of Contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,122 @@
...
+Recipes will vary in complexity.
+Some may be single inference calls.
+Others may illustrate useful agentic workflows.
+
+A "cookbook" is not intended to be a comprehensive guide to all
+issues that may arise during development with Granite Timeseries.
+Recipes will link to helpful external resources on topics including: distributed systems, UI, design, AI/ML theory, metrics, etc.
+
+### Exceptions
+
```
Comment (deanwampler) [inline README.md:64]: Should some of the TS content be in a separate README for the TS recipes?

**Review by deanwampler — APPROVED** [review]
One comment, but otherwise looks good.

File: README.md (hunk 1 above)
Comment (fayvor) [reply README.md:64]: Possibly; what TS content do you have in mind?

Comment (fayvor) [reply README.md:64]: @deanwampler I'm merging this PR and will address your comment in a later PR, once I clarify what you are looking for.

**Pull request merged**